ee/desktop/runner/runner.go

```diff
@@ -200,6 +210,8 @@ func (r *DesktopUsersProcessesRunner) Execute() error {
 	defer updateTicker.Stop()
 	menuRefreshTicker := time.NewTicker(r.menuRefreshInterval)
 	defer menuRefreshTicker.Stop()
+	osUpdateCheckTicker := time.NewTicker(5 * time.Second)
```

Comment:
Maybe a little less frequently than this? I could see once a minute being tolerable for the end user

Reply:
bumped to 1 min